tests/connection_e2e_fcr2gcp_test.go

```diff
@@ -0,0 +1,21 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCloudRouter2GcpCreateConnection(t *testing.T) {
+	// retryable errors in terraform testing.
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/fabric-cloud-router/fcr2gcp",
```

Comment:
Update path to "../examples/fabric/v4/cloudRouterConnectivity/cloudRouter2gcp"

Reply:
Ack.